packages/components/src/form-token-field/styles.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+/**
+ * Internal dependencies
+ */
+import { space } from '../ui/utils/space';
+
+type TokensAndInputWrapperProps = {
+	__next36pxDefaultSize: boolean;
+	hasTokens: boolean;
+};
+
+const deprecatedPaddings = ( {
+	__next36pxDefaultSize,
+	hasTokens,
+}: TokensAndInputWrapperProps ) =>
+	! __next36pxDefaultSize &&
+	css`
+		padding-top: ${ space( hasTokens ? 1 : 0.5 ) };
+		padding-bottom: ${ space( hasTokens ? 1 : 0.5 ) };
+	`;
+
+export const TokensAndInputWrapper = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	align-items: center;
+	gap: ${ space( 1 ) };
```

Comment:
Is there any reason why you prefer CSS styles to using the `Flex` and `HStack` components?

Reply:
Lol I forgot that existed! Replaced in 7e411f1.

I noticed that `<Flex>` does some magical `margin` computations under the hood to replicate `gap` in older browsers. Maybe we can remove that soon now that Safari is compliant.